Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml

```diff
@@ -0,0 +1,879 @@
+        "position": {
+          "x": 750,
+          "y": 260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 81ca3384-2d21-427b-800b-cf5afc21d27f
+    type: regular
+    task:
+      id: 81ca3384-2d21-427b-800b-cf5afc21d27f
+      version: -1
+      name: Search for suspicious-related alerts by MITRE Technique
+      description: |-
+        This task searches for Cortex XSIAM suspicious alerts related to the current incident by Mitre Technique, indicating that the alert is part of an attack pattern.
+
+        Focus on identifying alerts associated with the following MITRE techniques:
+        - Any Agent Alerts within this incident.
+        - T1059 - Command and Scripting Interpreter.
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      query:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '((mitreattcktechnique:*T1509* or (sourceBrand:TRAPS and categoryname:Malware)) and caseid:'
+              suffix:
+                value:
+                  simple: )
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1570,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 337f923f-d491-41ec-8d15-4034943b5796
+    type: condition
+    task:
+      id: 337f923f-d491-41ec-8d15-4034943b5796
+      version: -1
+      name: Is this a non-legit operation?
+      description: "## Determines if the LDAP operation was suspicious by the following conditions:\n\n- *Without Risky Asset Score:*  \nActor Process name is not Prevalent *AND*\nActor Process Command Line is not Prevalent *AND*\nNo Risk Asset Score.\n\n- *With Risky Asset Score:*  \n[Actor Process name is not Prevalent *OR*\nActor Process Command Line is not Prevalent] *AND*\nThe Host Risk Score is Medium or High or the User Risk Score is High."
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      Yes - With Risky Asset Score:
+      - "19"
+      yes - without Risky Asset Score:
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: yes - without Risky Asset Score
+      condition:
```

Comment:
For me - this condition has a bug.

You check: 
Process prevalence = false 
AND
cmd prevalence = false
AND 
Empty user score
AND 
Empty host score

So you have no suspicious parameter here but you still go to remediation

That needs a fix 

Am I wrong?

Reply:
the idea here is that if the operation is not prevalent from the process name and its cmd - we will go to the terminate process. why do you consider this a bug?

Reply:
cause you are making unneeded checks

Reply:
split this in to 2 conditions